roles/frontend/templates/frontend.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Frontend
+Requires=rest-api.service
```

Comment:
Your option will not work if the services are running on different instances
```suggestion
Wants=rest-api.service
```

Reply:
yes. my mistakes